riskiq-whoisiq/info.json

```diff
@@ -0,0 +1,861 @@
+        "results": "",
+        "domains": [
+          {
+            "domain": "",
+            "registrar": "",
+            "whoisServer": "",
+            "registered": "",
+            "registryUpdatedAt": "",
+            "expiresAt": "",
+            "contactEmail": "",
+            "nameServers": [
+            ],
+            "registrant": {
+              "domain": "",
+              "email": "",
+              "name": "",
+              "organization": "",
+              "street": "",
+              "city": "",
+              "state": "",
+              "postalCode": "",
+              "country": "",
+              "telephone": ""
+            },
+            "admin": {
+              "domain": "",
+              "email": "",
+              "name": "",
+              "organization": "",
+              "street": "",
+              "city": "",
+              "state": "",
+              "postalCode": "",
+              "country": "",
+              "telephone": ""
+            },
+            "billing": {
+              "domain": "",
+              "email": "",
+              "name": "",
+              "organization": "",
+              "street": "",
+              "city": "",
+              "state": "",
+              "postalCode": "",
+              "country": "",
+              "telephone": ""
+            },
+            "tech": {
+              "domain": "",
+              "email": "",
+              "name": "",
+              "organization": "",
+              "street": "",
+              "city": "",
+              "state": "",
+              "postalCode": "",
+              "country": "",
+              "telephone": ""
+            },
+            "zone": {},
+            "text": "",
+            "lastLoadedAt": ""
+          }
+        ]
+      },
+      "parameters": [
+        {
+          "title": "Name Server",
+          "name": "nameserver",
+          "tooltip": "Specify the name server you wish to search for..",
+          "description": "Specify the name server you wish to search for.",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "text"
+        },
+        {
+          "title": "Exact Match",
+          "name": "exact",
+          "tooltip": "Search for an exact match. Valid values are true and false.",
+          "description": "Search for an exact match. Valid values are true and false.",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "checkbox"
+        },
+        {
+          "title": "Maximum Results",
+          "name": "maxResults",
+          "tooltip": "Maximum number of results to return. Defaults to 1000 and maximum value is 10000..",
+          "description": "Maximum number of results to return. Defaults to 1000 and maximum value is 10000..",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "validation": {
```

Comment:
when we enter teh jinja expression the validation will fail. So please remove this. Handle the validation in the code.

Reply:
I have changed the type to integer and validation is working as expected

Reply:
ok